### PR TITLE
Jajacolonypolicy

### PR DESCRIPTION
**Pull request by jarpinosynthace**

### CODE REVIEW

**Comment by jarpinosynthace** [comment]
I changed the colony LH policy so that the ASP_WAIT setting is 0 as no need to wait, speeds up colony picking slightly

File: microArch/driver/liquidhandling/makelhpolicy.go [hunk 1]
```diff
@@ -424,7 +424,7 @@ func MakeColonyPolicy() LHPolicy {
 	policy["DSPZOFFSET"] = 0.0
 	policy["ASPSPEED"] = 3.0
 	policy["DSPSPEED"] = 3.0
-	policy["ASP_WAIT"] = 1.0
```
Comment (chr1sgrant) [inline microArch/driver/liquidhandling/makelhpolicy.go]: What's the purpose of this change? 
Speed increase?

Comment (jarpinosynthace) [inline microArch/driver/liquidhandling/makelhpolicy.go]: I changed the ASP_Wait setting from 1 to 0 to speed up colony picking. It was stabbing into the agar then waiting for a second before moving again. But its not necessary.

**Comment by chr1sgrant** [comment]
Can this be closed?

**Comment by jarpinosynthace** [comment]
Yea i guess it can be closed, the change i made was never merged in. But as we are not following up with the colony picking work for now maybe we can just ditch this.

**Comment by chr1sgrant** [comment]
I can also just merge it?

**Comment by chr1sgrant** [comment]
colony picking could come back

**Comment by jarpinosynthace** [comment]
Yea then merge it

**Comment by haydnKing** [comment]
closed as stale
